### PR TITLE
fix: fixed ubuntu installation step 2

### PR DESCRIPTION
**Pull request by ginruh**
Addresses #1533

### CODE REVIEW

**Comment by ginruh** [comment]
It's the same command but now can run without permission errors.

File: docs/installation/methods.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ tools:
       install:
         - curl -SsL https://packages.httpie.io/deb/KEY.gpg | sudo gpg --dearmor -o /usr/share/keyrings/httpie.gpg
       # - curl -SsL -o /etc/apt/sources.list.d/httpie.list https://packages.httpie.io/deb/httpie.list
-        - sudo echo "deb [arch=amd64 signed-by=/usr/share/keyrings/httpie.gpg] https://packages.httpie.io/deb ./" > /etc/apt/sources.list.d/httpie.list
+        - sudo sh -c 'echo "deb [arch=amd64 signed-by=/usr/share/keyrings/httpie.gpg] https://packages.httpie.io/deb ./" > /etc/apt/sources.list.d/httpie.list'
```
[the remark this answers is not in the document]
Comment (horw) [reply docs/installation/methods.yml:42]: No, just my comment here. These commands are almost equal.

[the remark this answers is not in the document]
Comment (Delta456) [reply docs/installation/methods.yml:42]: Just recommend both of them.

**Comment by mloskot** [comment]
This PR seems solid and ready, can this be merged?

File: docs/installation/methods.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hholst80) [reply docs/installation/methods.yml:42]: The tee variant is most common. Note that it Should not be -a to be equivalent with the original meaning.

**Comment by jkbrzt** [comment]
Thanks, @ginruh! Fixed via #1557.

**Pull request closed without merging**